components/mediation-message-processor/src/main/java/org/wso2/carbon/mediation/message/processor/cleanup/MessageProcessorCleanupTask.java

```diff
@@ -0,0 +1,45 @@
+package org.wso2.carbon.mediation.message.processor.cleanup;
+
+import org.apache.synapse.message.processor.MessageProcessor;
+import org.apache.synapse.message.processor.impl.ScheduledMessageProcessor;
+import org.wso2.carbon.base.MultitenantConstants;
+import org.wso2.carbon.mediation.initializer.services.SynapseEnvironmentService;
+import org.wso2.carbon.message.processor.util.ConfigHolder;
+import org.apache.synapse.message.processor.MessageProcessorCleanupService;
+
+public class MessageProcessorCleanupTask implements MessageProcessorCleanupService {
+    private static final long serialVersionUID = 1L;
+
+    private String name;
+
+    public Void call() throws Exception {
```

Comment:
Since we don't need to return anything from this method. We have used type parameters here. Therefore if we don't return anything we have to specify that explicitly.